tests/python/tests/test_trino.py

```diff
@@ -140,3 +140,28 @@ def tets_create_view(trino, warehouse: conftest.Warehouse):
     )
     r = cur.execute("SELECT * FROM test_create_view_trino.my_view").fetchall()
     assert r == [("a",), ("b",)]
+
+
+def tets_replace_view(trino, warehouse: conftest.Warehouse):
```

Comment:
```suggestion
def test_replace_view(trino, warehouse: conftest.Warehouse):
```
The name is actually important here, otherwise pytest doesn't pick it up.